### PR TITLE
Add visualization topic examples

### PR DESCRIPTION
**Pull request by jdbaldry**
Closes https://github.com/grafana/writers-toolkit/issues/560

Replaces https://github.com/grafana/writers-toolkit/pull/734

### CODE REVIEW

**Review by imatwawana — CHANGES_REQUESTED** [review]
My intent here was to actually have a screenshot. I'll update this when I have some bandwidth next week.

**Comment by jdbaldry** [comment]
If it changes your mind, this is consistent with the other topics documentation:

- https://grafana.com/docs/writers-toolkit/structure/topic-types/task/#task-topic-examples
- https://grafana.com/docs/writers-toolkit/structure/topic-types/reference/#reference-topic-examples
- https://grafana.com/docs/writers-toolkit/structure/topic-types/concept/#concept-topic-examples

File: docs/sources/structure/topic-types/visualization/index.md [hunk 1]
```diff
@@ -144,6 +144,14 @@ To write a visualization or widget topic, follow these steps.
 
    For more information about front matter, refer to [Front matter](https://grafana.com/docs/writers-toolkit/write/front-matter/).
 
+## Visualization topic examples
+
+Refer to the following topics for visualization topic examples:
+
+- [Alert list](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/alert-list/)
+- [Node graph](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/node-graph/)
+- [Time series](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/time-series/)
```
Comment (imatwawana) [inline docs/sources/structure/topic-types/visualization/index.md:153]:
```suggestion
- [Alert list](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/alert-list/)
- [Heatmap](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/heatmap/)
- [Stat](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/stat/)
```

**Review by imatwawana — APPROVED** [review]
I updated the list to use pages that follow the example to a tee

**Pull request merged**